code/game/atoms_movable.dm

```diff
@@ -1071,11 +1094,41 @@
 ///allows this movable to hear and adds itself to the important_recursive_contents list of itself and every movable loc its in
 /atom/movable/proc/become_hearing_sensitive(trait_source = TRAIT_GENERIC)
 	if(!HAS_TRAIT(src, TRAIT_HEARING_SENSITIVE))
-		RegisterSignal(src, SIGNAL_REMOVETRAIT(TRAIT_HEARING_SENSITIVE), .proc/on_hearing_sensitive_trait_loss)
+		//RegisterSignal(src, SIGNAL_REMOVETRAIT(TRAIT_HEARING_SENSITIVE), .proc/on_hearing_sensitive_trait_loss)
```

Comment:
commented code. Actually why is it commented?

Reply:
Unsure. It's in the original PR.

Reply:
we do a little trolling

Reply:
was just an oversight as i was working around with important_recursive_contents setters. it can be deleted

Reply:
Got it, thanks!